backend/activity_log/api_views.py

```diff
@@ -55,9 +58,23 @@ class EventTypeViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.EventTypeSerializer
     queryset = models.EventType.objects.order_by("name")
     filter_backends = [DjangoFilterBackend]
-
+    gte = datetime.now() + timedelta(days=7)
```

Comment:
a bit confused by why the gte is set to 7 days in the future. Isn't the idea that these need to be inputs, so we wouldn't want to hard-code values here?